qt/scientific_interfaces/Direct/ALFAnalysisView.cpp

```diff
@@ -190,6 +192,22 @@ void ALFAnalysisView::setupPeakCentreWidget(QGridLayout *layout, double const ce
   setPeakCentreStatus("");
 
   layout->addWidget(m_fitStatus, 3, 4);
+
+  // Add an empty label to act as empty space
+  layout->addWidget(new QLabel(""), 4, 4);
+}
+
+void ALFAnalysisView::setupRotationAngleWidget(QGridLayout *layout) {
+  m_rotationAngle = new QLineEdit("-");
+  m_rotationAngle->setReadOnly(true);
+  m_rotationAngle->setToolTip(ROTATION_ANGLE_TOOLTIP);
+  m_fitRequired = new QLabel("*");
+  m_fitRequired->setToolTip("A Fit to find the peak centre is required.");
+  m_fitRequired->setStyleSheet("QLabel { color: red; }");
```

Comment:
The asterisk appears quite small when testing. Could it be a little bigger? If this is consistent with the rest of the interface, then don't worry.
![image](https://user-images.githubusercontent.com/35813666/206725625-3825e7cc-ac5f-4997-a63a-770d5acd3dec.png)

Reply:
Thanks for reviewing! I think for this particular case I want the User not to notice the asterisk unless they are looking directly at the "Rotation" box and wondering why there isn't a value - so a small asterisk might be preferred

Also I think if we change the font-size of this individual asterisk then it won't respond to the User changing the font-size from the settings GUI. In general, I think the size is consistent with other interfaces as red asterisks are sized based on the font-size

Reply:
Makes sense, thanks for the additional context 👍